### PR TITLE
Add type: module to fix use in node

### PR DESCRIPTION
**Pull request by pearofducks**
postcss-cli hates the shit out of ESM, so I made a manual build 🤷 

To be clear we need to change to `type: module` because the component-classes are written in ESM.

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      'focus-visible-pseudo-class': false,
+    },
+  }),
+  importSvg({
+    paths: [iconsLocation],
+  }),
+  autoprefixer,
+  cssnano({ preset: 'default' })
+]
+const result = await postcss(plugins).process(css, { from, to })
```
Comment (digitalsadhu) [inline build.js:37]: Does calling postcss programmatically like this still use postcss.config.js?

**Review by digitalsadhu — COMMENTED** [review]
You'll need to reword your commits if you want to release a version. Or maybe this change is purely a chore and doesn't need a specific release?

File: build.js (hunk 1 above)
Comment (pearofducks) [reply build.js:37]: It does not (to my knowledge)

**Comment by pearofducks** [comment]
I'm planning to squash and merge, so no need to reword commits, right?

**Comment by digitalsadhu** [comment]
> I'm planning to squash and merge, so no need to reword commits, right?

Ah, so long as there's at least 1 commit that triggers a release such as fix: or feat: or a body with BREAKING CHANGE in it, it should trigger a release. Everything else won't but might be used in a changelog. Will your "Fix" commit trigger?

**Comment by pearofducks** [comment]
The commit message going back into main will be `fix: use ESM to unbreak component-classes`

**Comment by pearofducks** [comment]
@digitalsadhu - any concerns with this merging?

**Comment by digitalsadhu** [comment]
Go for it

**Pull request merged**